conf/nginx/nginx.conf.docs.common.template

```diff
@@ -0,0 +1,20 @@
+!{explode server(docs)}
+
+# static files
+location ^~ /docs/${server_id}/loleaflet {
+    rewrite /docs/${server_id}/(.*) /$1 break;
+    proxy_pass http://docs-${server_id};
```

Comment:
Ideally we're using TLS everywhere by default, so our default setup should use https, not http

Reply:
The service will be exposed to the client via https (if enabled on nginx), we are talking about internal communication inside the cluster: changes in the document as well a rendered document pass through this channel.
Can we introduce the SSL in a future iteration? it's quite tricky to introduce the ssl on this channel, the main concern is how to make available the certificate (possibly self-signed) to nginx.

Reply:
Understood. We already to TLS internally by default (even with self-signed certs) for all other services, doing non-TLS by default for a new feature would be a regression in my mind. I vote we do this right the first time.